backend/app/models/book.rb

```diff
@@ -0,0 +1,2 @@
+class Book < ApplicationRecord
```

Comment:
I suggest using the gem `annotate`. It automatically updates the `model` files with their `table` columns. Easy reference to know the shape of a model's table.